### ISSUE TITLE
Strongly suggest rename

### ISSUE THREAD

**Issue by mmontag** [Lunakepio/Mario-Kart-3.js#6]
If you have serious ambitions for the project, and for the sake of potential contributors, I urge maintainers to consider a slightly different name. You don't want to attract the wrong kind of attention...

**Comment by GilDev** [comment]
That is actually already planned: [Pup Cup is coming](https://www.linkedin.com/posts/alex-moulinneuf-3a77b61b6_javascript-creativecoding-developpeurweb-activity-7159850916987158528-KGER?utm_source=share&utm_medium=member_desktop) 🙂

**Comment by Lunakepio** [comment]
Indeed, don't worry there's no legal issue for having Mario Kart in the repository name, Pup Cup is still coming

**Comment by david8128** [comment]
Hey @Lunakepio, just a heads up, let's keep our repo going strong and trouble-free for as long as we can!

**Comment by ga-sfr** [comment]
Will Pup Cup be multiplayer like Smash Karts?
